packages/@aws-cdk/aws-appmesh/lib/virtual-gateway.ts

```diff
@@ -180,7 +179,9 @@ export class VirtualGateway extends VirtualGatewayBase {
       meshName: this.mesh.meshName,
       spec: {
         listeners: this.listeners.map(listener => listener.listener),
-        backendDefaults: props.backendsDefaultClientPolicy?.bind(this),
+        backendDefaults: props.backendDefaults !== undefined ? {
+          clientPolicy: props.backendDefaults?.clientPolicy?.bind(this).clientPolicy,
+        } : undefined,
```

Comment:
Can we indent this slightly differently?

```suggestion
        backendDefaults: props.backendDefaults !== undefined
          ? {
            clientPolicy: props.backendDefaults?.clientPolicy?.bind(this).clientPolicy,
          }
          : undefined,
```